### PR TITLE
Fixed broken link in gh-actions badge

### PR DESCRIPTION
**Pull request by jlstevens**
Not sure what the intended link was but it isn't working right now so I'm proposing what I think was supposed to be the target...

### CODE REVIEW

**Comment by jlstevens** [comment]
Hvplot uses: `https://github.com/holoviz/hvplot/actions?query=workflow%3Atests`

**Comment by philippjfr** [comment]
Thanks, I think they actually changed the URL at some point.

**Comment by Hoxbro** [comment]
Would it not be better to have it referencing the master branch? This can be done by appending`?query=branch%3Amaster` to both urls.

**Comment by philippjfr** [comment]
Good suggestion @Hoxbro. If you have the time @jlstevens we should do that everywhere.

**Comment by jlstevens** [comment]
> ... we should do that everywhere.

I'm happy to update our main projects (and list the ones that I updated) but I'm not sure about *everywhere* as such. :-)

**Review by jbednar — APPROVED** [review]

**Review by philippjfr — CHANGES_REQUESTED** [review]
Will merge once this links to the master branch as Hoxbro suggested.

File: README.md [hunk 1]
```diff
@@ -6,7 +6,7 @@
 
 |    |    |
 | --- | --- |
-| Build Status | [![Linux/MacOS Build Status](https://github.com/holoviz/panel/workflows/pytest/badge.svg)](https://github.com/holoviz/panel/workflows/pytest)
+| Build Status | [![Linux/MacOS Build Status](https://github.com/holoviz/panel/workflows/pytest/badge.svg)](https://github.com/holoviz/panel/actions/workflows/test.yaml)
```
Comment (philippjfr) [inline README.md:9]:
```suggestion
| Build Status | [![Linux/MacOS Build Status](https://github.com/holoviz/panel/workflows/pytest/badge.svg?query=branch%3Amaster)](https://github.com/holoviz/panel/actions/workflows/test.yaml?query=branch%3Amaster)
```

**Review by philippjfr — APPROVED** [review]

**Pull request merged**